### PR TITLE
feature(new feature): Adjust- adding partner parameters

### PR DESCRIPTION
**Pull request by rohithkaza**
## Description of the change

> Adding support for partner parameters

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/adj/transform.js [hunk 1]
```diff
@@ -44,6 +44,23 @@ function responseBuilderSimple(message, category, destination) {
     response.headers = {
       Accept: "*/*"
     };
+
+    const partnerParamsKeysMap = getHashFromArray(
+      destination.Config.partnerParamsKeys
```
Comment (ItsSudip) [inline v0/destinations/adj/transform.js:49]:
```suggestion
      destination?.Config?.partnerParamsKeys
```

File: v0/destinations/adj/transform.js [hunk 2]
```diff
@@ -44,6 +44,23 @@ function responseBuilderSimple(message, category, destination) {
...
+    );
+    if (partnerParamsKeysMap) {
+      payload.partner_params = {};
+      Object.keys(partnerParamsKeysMap).forEach(key => {
+        if (message.properties[key]) {
+          payload.partner_params[partnerParamsKeysMap[key]] =
+            message.properties[key];
```
Comment (ItsSudip) [inline v0/destinations/adj/transform.js:56]:
```suggestion
            message.properties[key].toString();
```
As adjust is only supporting strings.

File: v0/destinations/adj/transform.js (hunk 1 above)
Comment (rohithkaza) [reply v0/destinations/adj/transform.js]: Done

File: v0/destinations/adj/transform.js (hunk 2 above)
Comment (rohithkaza) [reply v0/destinations/adj/transform.js]: Done

**Review by ItsSudip — DISMISSED** [review]

**Review by utsabc — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**